client/src/Movies/MovieList.js

```diff
@@ -28,24 +31,11 @@ const MovieList = props => {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
+  const { title, director, metascore, stars, id } = movie;
```

Comment:
Good job refactoring code to be dry.  Another approach you could have taken was to use MovieCard inside MovieList and just delete MovieDetails